ingestion/functions/common/common_lib.py

```diff
@@ -85,11 +89,28 @@ def complete_with_error(
     """
     print(exception)
     if env and upload_error and source_id and upload_id:
-        finalize_upload(env, source_id, upload_id, headers,
+        finalize_upload(env, source_id, upload_id, headers, cookies,
                         error=upload_error)
     raise exception
 
 
+def login(email: str):
+    """Logs-in a local curator server instance for testing.
+
+    Returns the cookie of the now logged-in user.
+    """
+    print('Logging-in user', email)
+    endpoint = "http://localhost:3001/auth/register"
```

Comment:
I'm undecided as to whether this is right, or whether we should use the utility function, and have `login` check that `env` == local.

Reply:
FWIW I am checking in the main that env == 'local' before calling login()

Reply:
Yeah, I don't think there's a practical difference either way. In fact, what you have might be better, since the common lib appends `/api` to the base URL. Only downside is port changes, but that feels pretty minor.